### ISSUE TITLE
PhoneAuthProvider sign-in not working in (CDN) 2.6

### ISSUE THREAD

**Issue by spoxies** [firebase/firebaseui-web#314]
Seems like the sign-in with the phone provider does not work in 2.6. (it does work in 2.5)

It throws an error see attached: 

![screen shot 2018-02-16 at 15 59 41](https://user-images.githubusercontent.com/20624094/36313662-788e06aa-1332-11e8-8470-626da1b722d5.png)

Error ends @82:
```
        function Ca(a, b, c) {
            return a.call.apply(a.bind, arguments)
        }
```
Error triggered from @323:
```
f ? (b.ua("mdl-spinner mdl-spinner--single-color mdl-js-spinner is-active firebaseui-progress-dialog-loading-icon", A("Verifying...").toString()),
```
Sample code working in **2.5** change to **2.6**:
```
<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8">
    <title>API sign-in</title>
    <script src="https://www.gstatic.com/firebasejs/4.10.0/firebase.js"></script>
    <script>
      var config = {
            //TODO Config here
      };
      firebase.initializeApp(config);
    </script>
    
    //Version 2.5 works, 2.6 does not
    <script src="https://cdn.firebase.com/libs/firebaseui/2.5.0/firebaseui.js"></script>
    <link type="text/css" rel="stylesheet" href="https://cdn.firebase.com/libs/firebaseui/2.5.0/firebaseui.css"> 
    
    <script type="text/javascript">
      var uiConfig = {
        callbacks: {
            'signInSuccess': function(user, credential, redirectUrl) {
              console.log(user);
              return false;
            }
        },
        signInOptions: [
            {
              provider: firebase.auth.PhoneAuthProvider.PROVIDER_ID,
              defaultCountry: 'EN',
              recaptchaParameters: {
                size: 'invisible'
              },
            }
          ]
      };
      var ui = new firebaseui.auth.AuthUI(firebase.auth());
      ui.start('#firebaseui-auth-container', uiConfig);

    </script>
  </head>
  <body>
    <h1>API sign-in</h1>
    <div id="firebaseui-auth-container"></div>
  </body>
</html>
```

**Comment by bojeil-google** [comment]
Thanks for reporting this @spoxies, I can reproduce it. I will work an fix asap.

**Comment by bojeil-google** [comment]
Thanks @spoxies.
The issue was in the firebase CDN built through the open source closure compiler. We only tested with the gstatic CDN build which was working fine. We will test for both going forward.
So if you had used that version of `2.6.0`, it would have worked.
This should be fixed in release `2.6.1`. Thanks again for notifying us and we apologize for the inconvenience.

**Comment by spoxies** [comment]
Thanks for the fast fix. Well done!
